docs/content/docs/recipes/react.md

```diff
@@ -120,60 +83,64 @@ const azureClientConfig = {
 1. Replace `TODO 1` with the following code. Note that there is only one object in the container: a SharedMap holding the timestamp. Note also that `sharedTimestamp` is the ID of the `SharedMap` object and it must be unique within the container.
 
     ```js
-    const { containerId, isNew } = getContainerId();
-    const azureClient = new AzureClient(azureClientConfig);
-    const containerSchema = {
-        name: 'fluid-react-tutorial-container',
-        initialObjects: { sharedTimestamp: SharedMap }
-    };
+      const client = new TinyliciousClient();
+      const containerSchema = {
+          initialObjects: { sharedTimestamp: SharedMap }
+      };
     ```
 
-1. Replace `TODO 2` with the following code. Note that `isNew` was returned by the `getContainerId` helper method and it is true if the application has no Fluid container yet.
+1. Replace `TODO 2` with the following code. Note that `containerId` is being stored on the URL hash, and if there is no `containerId` we create a new container instead.
 
     ```js
-    const { fluidContainer } = isNew
-        ? await azureClient.createContainer({id: containerId}, containerSchema)
-        : await azureClient.getContainer({id: containerId}, containerSchema);
+    let container;
+    const containerId = location.hash.substring(1);
+    if (!containerId) {
+        ({ container } = await client.createContainer(containerSchema));
+        const id = await container.attach();
+        location.hash = id;
+    } else {
+        ({ container } = await client.getContainer(containerId, containerSchema));
+    }
     ```
 
 1. Replace `TODO 3` with the following code.
 
     ```js
-    return fluidContainer.initialObjects;
+    return container.initialObjects;
     ```
 
 ### Get the Fluid data on application startup
 
 Now that we've defined how to get our Fluid data, we need to tell React to call `getFluidData` when the application starts up and then store the result in state. So add the following code at the top of the `App()` function (above the `return` statement). Note about this this code:
 
 - By setting an empty dependency array at the end of the useEffect, we ensure that this function only gets called once.
-- Since `setFluidSharedMap` is a state-changing method, it will cause the React `App` component to immediately rerender.
+- Since `setFluidSharedObjects` is a state-changing method, it will cause the React `App` component to immediately rerender.
 
 ```js
-const [fluidSharedMap, setFluidSharedMap] = React.useState();
+const [fluidSharedObjects, setFluidSharedObjects] = React.useState();
```

Comment:
the original tutorial mislabeled the returned "defaultObjects" as a SharedMap, which while technically true, there was later confusion about where events originated, so I changed this state variable to be more explicit, and focused on the sharedTimeStamp as the important ``SharedMap`